### DISCUSSION TITLE
特定のキーが動かない

### DISCUSSION THREAD

**Discussion by nanon-key** [Yowkees/keyball#211] [General]
@Yowkees 

夜分遅くに失礼します。
初めての自作キーボード楽しく作らさせていただきました。
ただ、あと完成まで、あと一歩のところでつまずいてしまいました。過去のやり取り、できる限り拝見いたしましたが、何か失礼があれば申し訳ないですが、ご質問させてください。

![42B0A0BA-D731-4B03-BEAF-B295F0A956A0](https://user-images.githubusercontent.com/130074326/230426358-1f5ff954-0425-4f2c-b63f-d26ebe29659e.jpeg)

こちらのキーが動かず困っています。
テスターをショートするやり方は試してみましたが、反応せずです。

何卒ご意見頂けますと幸いです。

**Comment by nanon-key** [comment]
追加、付近のダイオードの向き等も確認済みです。

**Comment by Yowkees** [comment]
ご連絡が遅くなり申し訳ありません。
下記画像の緑丸のダイオードが該当しますので、念のため再度ご確認ください。
また、COL5の信号線がうまくつながっていなくても同様の症状となります。緑矢印の端子です。
このProMicro側の半田付けがうまくいっているかご確認お願いします。
また、オレンジ矢印部分がショートしていないかもご確認ください。
お手数ですがご対応よろしくお願いいたします。
![Keyball39-fix](https://user-images.githubusercontent.com/84209772/231072004-5d7c7dfb-410e-47ec-b03f-fb7423e203b3.jpg)
